OpenSim/Simulation/Model/OffsetFrame.h

```diff
@@ -46,6 +46,11 @@ namespace OpenSim {
  * OffsetFrame in the tree. This allows Solvers and algorithms to work directly
  * with the Base which can be more efficient.
  *
+ * OffsetFrame is an abstract class. Derive concrete subclasses in order to
+ * accommodate new Frame types that require their offsets to retain the same
+ * type as the parent. For example:
+ *  class PhysicalOffsetFrame : public OffsetFrame<PhysicalFrame>
```

Comment:
In HTML, this will look like regular text. Is that your intent? Perhaps put `@code`, `@endcode`?